iosApp/iosApp/AppDelegate.swift

```diff
@@ -11,17 +11,41 @@ import ConfettiKit
 
 class AppDelegate : NSObject, UIApplicationDelegate {
     
-    let root: AppComponent
```

Comment:
Maybe we could conform `AppDelegate` to `ObservableObject` for simplicity? And just make this `root` property `@Published var`. Can you try that? If for some reason this is not possible or not working, then maybe move the `AppComponent` creation and replacing logic to `AppComponentHolder`?

Otherwise looks good!

Reply:
Hmm yeah I didn't try to make `AppDelegate` an `ObservableObject` since I wasn't sure if that would be a weird thing to be that and an `UIApplicationDelegate` at the same time. I will give that a shot.

If that does not work for whatever reason, when you say "maybe move the AppComponent creation and replacing logic to AppComponentHolder" I assume you mean so that `AppDelegate` just calls a small function in `init()` and on `onConferenceDeepLink` so that the bulk of the work is done in there instead.

Will pick this up some next week, hopefully earlier rather than later inside of it. Thanks for the feedback 😊

Reply:
>  when you say "maybe move the AppComponent creation and replacing logic to AppComponentHolder"

Yes. Just create the `AppComponent` together with its `ComponentContext` and `Lifecycle` inside `AppComponentHolder`. And also add `onConferenceDeepLink` to `AppComponentHolder` instead, and call that from `AppDelegate`. I think this would be cleaner. But the first option with conforming `AppDelegate` to `ObservableObject` sounds even better.

Reply:
Hey making AppDelegate conform to ObservableObject just worked, thanks for your tips!